src/preload.py

```diff
@@ -189,12 +188,30 @@ def start_docker_daemon(storage_driver, mountpoint):
     return running_dockerd
 
 
+def read_file(name):
+    with open(name, "rb") as f:
+        return f.read()
+
+
+def write_file(name, content):
+    with open(name, "wb") as f:
+        f.write(content)
+
+
 @contextmanager
 def docker_context_manager(storage_driver, mountpoint):
-    running_dockerd = start_docker_daemon(storage_driver, mountpoint)
+    docker_dir = fix_rce_docker(mountpoint)
+    # If we don't remove <part6>/<docker|rce>/network/files/local-kv.db and the
+    # preload container was started with bridged networking, the following
+    # dockerd is not reachable from the host.
+    local_kv_db_path = "{}/network/files/local-kv.db".format(docker_dir)
+    local_kv_db_content = read_file(local_kv_db_path)
+    os.remove(local_kv_db_path)
```

Comment:
@petrosagg do you know why this file prevents in inner dockerd from being reachable from the host if the container is started with bridged networking?